apiv2/template_overrides/support/enums.py.j2

```diff
@@ -0,0 +1,19 @@
+"""
+GraphQL enums
```

Comment:
I forgot to commit this file earlier, but the changes it makes (converting spaces to underscores in enum values) are already in the `main` branch.